### DISCUSSION TITLE
Task properties/metadata

### DISCUSSION THREAD

**Discussion by azzamsa** [BaldissaraMatheus/Tasks.md#129] [Ideas]
Hi.

Can we have something like `created time`, `last modified time`, and `finished time` embedded in the task?
With this properties/metadata feature added, we can also embed many things, such as tags, contexts, priorities, etc.

Thanks! 🥞

**Comment by BaldissaraMatheus** [comment]
About the dates, there's something similar asked on #84, and I was actually considering storing this information within the file's metadata. So yeah, that's actually something I'm already planning to do. I was also considering moving the tags to metadata as well (today they are kept within the file itself)

**Reply by kitforbes** [reply]
I've just started taking a look at the project and thought that it would be useful to add metadata like this to YAML front-matter within each file. For example:
```markdown
---
title: Fix the broken thing
created: 2025-08-24
tags: bug, frontend
---

That broken thing on the site should be fixed.
```
`tags` could replace (or suppliment) the current `[tag:bug]` syntax and `created` could optionally be rendered on the task. `title` could be used as the "task" title in the UI if it's present.

This may be more than what you're looking for in this project though. If you're interested I can look at making some additions.
